scripts/all-utilities.sh

```diff
@@ -310,11 +310,15 @@ function get_serverless_image_name() {
 }
 
 function get_proxy_image_release_name() {
-  echo -n 'gcr.io/endpoints-release/endpoints-runtime'
+  echo -n "gcr.io/${ESP_RELEASE_PROJECT}/endpoints-runtime"
 }
 
 function get_serverless_image_release_name() {
-  echo -n 'gcr.io/endpoints-release/endpoints-runtime-serverless'
+  echo -n "gcr.io/${ESP_RELEASE_PROJECT}/endpoints-runtime-serverless"
+}
+
+function get_gcsrunner_image_release_name() {
+  echo -n "gcr.io/${ESP_RELEASE_PROJECT}/gcsrunner"
```

Comment:
can we keep it in our cloudese-testing project for gcsrunner?

endpoints-release is for users.  gcsrunner is not for users.

Reply:
Xuyang made a good comment offline that `cloudesf-testing` would also have all the CI builds in it which might get messy. If we don't want to put it in `endpoints-release`, would it be preferable for us to have a separate GCR repo for the gcsrunner releases?

Reply:
Yes, we can do that. The destination project totally depends on you guys.

Reply:
Okay, let me talk with the team to see what we want it to be named.

Reply:
Updated to be ` gcr.io/espv2-gcsrunner-release/gcsrunner`

PTAL